### PR TITLE
Small fix

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

File: code/controllers/subsystem/npcpool.dm [hunk 1]
```diff
@@ -19,12 +19,13 @@ SUBSYSTEM_DEF(npcpool)
...
 	..("NPCS:[processing.len]|D:[needsDelegate.len]|A:[needsAssistant.len]|U:[canBeUsed.len]")
 
-/datum/controller/subsystem/npcpool/proc/stop_processing(mob/living/carbon/human/interactive/I)
+/datum/controller/subsystem/npcpool/proc/stop_processing(mob/living/I)
 	processing -= I
 	currentrun -= I
 	needsDelegate -= I
 	canBeUsed -= I
 	needsAssistant -= I
+	currentrun -= I
```
Comment (lzimann) [inline code/controllers/subsystem/npcpool.dm]: It's doing `currentrun -= I` twice

**Comment by Cyberboss** [comment]
Am I drank'

**Comment by Cyberboss** [comment]
The Avatar is yes

**Pull request merged**